appveyor.yml

```diff
@@ -27,7 +27,7 @@ build_script:
   - cmake \
     -G"Visual Studio 15 2017 Win64" \
     -DCMAKE_CONFIGURATION_TYPES=%CONFIGURATION% \
-    -DCMAKE_INSTALL_PREFIX=%APPVEYOR_BUILD_FOLDER%\install \
+    -DCMAKE_INSTALL_PREFIX=%APPVEYOR_BUILD_FOLDER%\cmake_install \
```

Comment:
`install` now conflicts with the top-level `INSTALL` file (formerly `openvdb/INSTALL`).

Reply:
The contents of the INSTALL file are a bit odd IMO. There's the dependency information which is useful, but less verbose than the information stored in doc/dependencies.txt and then there's the Makefile build instructions, which I feel should at least live alongside openvdb/Makefile rather than being moved up into the parent directory where there is no Makefile.

Perhaps we consider moving the Makefile build instructions into doc/build.txt and then either removing all the dependency information in favour of a direct website link from README.md or otherwise append it to that readme in a new dependency section?

Reply:
Agreed.  The INSTALL file has outlived its usefulness.